.github/workflows/pull-request-checks.yaml

```diff
@@ -352,7 +352,7 @@ jobs:
         run: ./.github/workflows/pull-request-check-clang-format.sh
 
   check-cpplint:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
```

Comment:
I'm slightly weary of downgrading CI platforms to something that was released over 3 years ago. Do we really need to do this?

Reply:
Reverted and enforcing the use of python3 for our internal script instead.